python/lsst/ip/isr/isrTask.py

```diff
@@ -102,6 +102,11 @@ class IsrTaskConfig(pexConfig.Config):
         doc = "Name of mask plane to use in saturation detection and interpolation",
         default = "SAT",
     )
+    suspectMaskName = pexConfig.Field(
+        dtype = str,
+        doc = "Name of mask plane to use for suspect pixels",
```

Comment:
Probably worth either incorporating or linking to the text you're adding to afw about what suspect means, here.